src/components/ff-linkify.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import URLFinder from 'ff-url-finder'
+import config from '../config'
+
+const MAX_URL_LENGTH = 50
+
+const LINK = 'link'
+const AT_LINK = 'atLink'
+const LOCAL_LINK = 'localLink'
+const EMAIL = 'email'
+
+const finder = new URLFinder(
+  ['ru', 'com', 'net', 'org', 'info', 'gov', 'edu', 'рф', 'ua'],
+  config.siteDomains,
+)
+
+class FFLinkify extends React.Component {
+  static propTypes = {
+    component: React.PropTypes.any,
+    properties: React.PropTypes.object,
+  }
+
+  static defaultProps = {
+    component: 'a',
+    properties: {},
+  }
+
+  parseCounter = 0
+
+  parseString(string) {
+    let elements = []
+    if (string === '') {
+      return elements
+    }
+
+    let idx = 0
+
+    finder.parse(string).map(it => {
+      let displayedLink
+      let href
+
+      if (it.type === LINK) {
+        displayedLink = URLFinder.shorten(it.text, MAX_URL_LENGTH)
+        href = it.url
+      } else if (it.type === AT_LINK) {
+        displayedLink = it.text
+        href = `/${it.username}`
```

Comment:
It seems it will highlight all the @links. It would be nice to only highlight existing usernames.